src/Voron/Platform/Posix/PosixMemoryMapPager.cs

```diff
@@ -206,6 +206,12 @@ public override void ReleaseAllocationInfo(byte* baseAddress, long size)
             NativeMemory.UnregisterFileMapping(FileName, ptr, size);
         }
 
+        public override void FillZeroLengthJournalWithZeros(ulong size)
```

Comment:
Will not actually work, the error is actually being thrown from the constructor.